easybuild/easyblocks/o/openfoam.py

```diff
@@ -486,6 +486,11 @@ def sanity_check_step(self):
             test_foammonitor = "! foamMonitor -h 2>&1 | grep 'not installed'"
             custom_commands.append(' && '.join([load_openfoam_env, test_foammonitor]))
 
+        if self.is_dot_com and self.looseversion >= LooseVersion("2012"):
```

Comment:
This should always have worked, but I could be conservative and only actually check for later versions of OpenFOAM.com